src/pages/search/SearchTypeField.tsx

```diff
@@ -4,21 +4,37 @@ import ShowChartIcon from '@mui/icons-material/ShowChart';
 import { MenuItem, TextField, TextFieldProps } from '@mui/material';
 import { useTranslation } from 'react-i18next';
 import { useLocation, useNavigate } from 'react-router';
+import { PersonSearchParameter, ProjectSearchParameter } from '../../api/cristinApi';
+import { ResultParam } from '../../api/searchApi';
 import { dataTestId } from '../../utils/dataTestIds';
-import { SearchParam } from '../../utils/searchHelpers';
+import { SearchParam, syncParamsWithSearchFields } from '../../utils/searchHelpers';
 
 export enum SearchTypeValue {
-  Result = 'result',
+  Result = 'registration',
   Person = 'person',
   Project = 'project',
 }
 
+const getSyncedQueryValue = (params: URLSearchParams, searchType: SearchTypeValue) => {
+  const syncedParams = syncParamsWithSearchFields(params);
```

Comment:
Den flyten vår med synced params er ikke helt åpenbar, men det er altså for å få tak i verdier brukeren har skrevet inn med ikke submitted.
Si fra om noen vil ha gjennomgang av hva/hvorfor/hvordan av dette!